PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,7 @@
 | Q               | A
 | --------------- | ---
-| Bug fix?        | no|yes
-| New feature?    | no|yes
-| BC breaks?      | no|yes
+| Bug fix?        | no\|yes
+| New feature?    | no\|yes
+| BC breaks?      | no\|yes
```

Comment:
But why? I want to choose "yes" or "no" and that's it, so using "/" is one less backspace.